.github/workflows/test-integration.yaml

```diff
@@ -31,18 +31,23 @@ jobs:
         # integration testing for 3.10 and 3.11 (for runner versions that follow that PR).
         python-version: ["3.9"]  # , "3.10", "3.11"]
         runner-version: [
-          "pangeo-forge-runner==0.8.0",
-          "pangeo-forge-runner==0.9.0",
+          "pangeo-forge-runner==0.9.1",
+          "pangeo-forge-runner==0.9.2",
         ]
     steps:
       - uses: actions/checkout@v4
-      - name: 🔁 Setup Python
-        id: setup-python
-        uses: actions/setup-python@v4
+
+      - name: Setup conda
+        uses: mamba-org/setup-micromamba@v1
         with:
-          python-version: ${{ matrix.python-version }}
-          cache: pip
-          cache-dependency-path: pyproject.toml
+          create-args: python==${{ matrix.python-version }}
+          environment-name: pangeo-forge
+
+      - name: Install pangeo-forge recipes and runner
+        shell: bash -l {0}
+        run: |
+          python -m pip install ${{ matrix.runner-version }}
+          python -m pip install -e ".[test,minio,grib]"
```

Comment:
If we are using pip to install GRIB dependencies, do we need to be using micro mamba? Can't we just keep Setup Python the way it was?

Reply:
Also, I'm somewhat surprised this works? The `cf-grib`/`eccodes` docs seems to suggest system-level dependencies need to be installed via conda?

Reply:
Good call, just reverted that and the integration tests are passing.